packages/web/components/TechnologyForm/MapAndAttachments/styles.js

```diff
@@ -70,7 +70,7 @@ export const Suggestion = styled.div`
 	padding: 1rem;
 	width: 100%;
 	font-size: 1.4rem;
-	cursor: pointer:
+	cursor: pointer;
```

Comment:
![image](https://user-images.githubusercontent.com/30202634/97246870-2e7bab00-17dd-11eb-9ff5-2f1b0dcd5589.png)

Reply:
It was already there, I forgot to fix, ty 😄